src/ast/nodes/ClassDeclaration.ts

```diff
@@ -27,6 +27,13 @@ export default class ClassDeclaration extends ClassNode {
 		if (options.systemBindings && this.id.variable.exportName) {
 			code.appendRight(this.end, ` exports('${this.id.variable.exportName}', ${this.id.variable.getName()});`);
 		}
+		if (this.id) {
+			const name = this.id.variable.getName()
+			if (name !== this.id.variable.name) {
+				code.appendRight(this.start, `let ${this.id.variable.safeName} = `)
+				code.prependRight(this.end, ';');
```

Comment:
This, on the other hand, needs to be `prependLeft` with the explanation given above.